### PR TITLE
Feature ftoz nec3

### PR DESCRIPTION
**Pull request by VikkyMaru**

### CODE REVIEW

File: Inception/ClientApp/src/App/Home/HomeComponent.html [hunk 1]
```diff
@@ -25,15 +25,16 @@
...
                         {{ domainName | RemoveScheme}}
                         </mat-option>
                     </mat-autocomplete>
             </mat-form-field>
             <button mat-button class="TestButton" (click)="ShowTestResultHistoryList()">Test me!</button>
             <button mat-button class="TestButton"
                 *ngIf="SiteTestResultThumbnails != null" 
-                [routerLink]="['/overview', DomainName]" >Site overview</button>
+                [routerLink]="['/overview', DomainName]" 
+                >Site overview</button>
```
Comment (PavelLev) [inline Inception/ClientApp/src/App/Home/HomeComponent.html]: Inconsistent style

File: Inception/ClientApp/src/App/Home/SiteTestOverviewComponent.ts [hunk 2]
```diff
@@ -10,13 +14,20 @@ import { Component, OnInit } from "@angular/core";
...
+    )
     {
-
+        this.DomainName =  this._activatedRoute.snapshot.params.domain;
     }
 
     public ngOnInit(): void
     {
 
+        this._siteTestOverviewService.GetSiteTestOverview(this.DomainName).subscribe(SiteTestOverview => this.SiteTestOverview = SiteTestOverview);
```
Comment (PavelLev) [inline Inception/ClientApp/src/App/Home/SiteTestOverviewComponent.ts]: Move subscribe to new line

**Review by VikkyMaru — COMMENTED** [review]
Great implementation (:

**Review by PavelLev — APPROVED** [review]
+1

**Pull request merged**